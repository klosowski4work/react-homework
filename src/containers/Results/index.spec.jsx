import React from 'react';
import renderer from 'react-test-renderer';
import { Results } from './';

describe('Results component', () => {
    it('should render component', () => {
        const component = renderer.create(<Results />).toJSON()
        expect(component).toMatchSnapshot();
    });
});