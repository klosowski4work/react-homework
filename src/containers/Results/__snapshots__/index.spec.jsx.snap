// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results component should render component 1`] = `
<div
  className="results"
>
  <div
    className="results__header"
  >
    <span>
      x movies found
    </span>
    <div
      className="sort-by"
    >
      <h3
        className="sort-by__title"
      >
        Sort by
      </h3>
      <button
        className="button-link   "
        onClick={[Function]}
        type="button"
      >
        release date
      </button>
      <button
        className="button-link   "
        onClick={[Function]}
        type="button"
      >
        rating
      </button>
    </div>
  </div>
  <div
    className="results__container"
  >
    <div
      className="movie"
    >
      <div
        className="movie__cover"
      >
        <img
          src=""
        />
      </div>
      <div
        className="movie__footer"
      >
        <div
          className="movie__title"
        >
          Test1
        </div>
        <div
          className="movie__type"
        >
          type
        </div>
        <div
          className="movie__release-date"
        >
          2018
        </div>
      </div>
    </div>
    <div
      className="movie"
    >
      <div
        className="movie__cover"
      >
        <img
          src=""
        />
      </div>
      <div
        className="movie__footer"
      >
        <div
          className="movie__title"
        >
          Test2
        </div>
        <div
          className="movie__type"
        >
          type
        </div>
        <div
          className="movie__release-date"
        >
          2018
        </div>
      </div>
    </div>
    <div
      className="movie"
    >
      <div
        className="movie__cover"
      >
        <img
          src=""
        />
      </div>
      <div
        className="movie__footer"
      >
        <div
          className="movie__title"
        >
          Test3
        </div>
        <div
          className="movie__type"
        >
          type
        </div>
        <div
          className="movie__release-date"
        >
          2018
        </div>
      </div>
    </div>
    <div
      className="movie"
    >
      <div
        className="movie__cover"
      >
        <img
          src=""
        />
      </div>
      <div
        className="movie__footer"
      >
        <div
          className="movie__title"
        >
          Test4
        </div>
        <div
          className="movie__type"
        >
          type
        </div>
        <div
          className="movie__release-date"
        >
          2018
        </div>
      </div>
    </div>
    <div
      className="movie"
    >
      <div
        className="movie__cover"
      >
        <img
          src=""
        />
      </div>
      <div
        className="movie__footer"
      >
        <div
          className="movie__title"
        >
          Test5
        </div>
        <div
          className="movie__type"
        >
          type
        </div>
        <div
          className="movie__release-date"
        >
          2018
        </div>
      </div>
    </div>
  </div>
</div>
`;
